### PR TITLE
Correct Spirit Healers position.

### PR DESCRIPTION
**Pull request by Daribon**
## 🍰 Pullrequest

Corrects all spirit healers positions based on sniff that weren't corrected previously.

### Proof

Sniff

### Issues

- None

### How2Test

- .go creature 40528
- .die
- Release spirit.
- Position should now match up with world_safe_locs_facing data.

### Todo / Checklist

After applying this sql, the only spirit healers that will not be in blizzlike postions are the following:
- [x] Deadwind Pass Spirit Healer 17649
- [x] Deathknell Spirit Healer 40547
- [x] Red Cloud Mesa Spirit Healer 40565
- [x] Coldridge Valley Spirit Healer 87046
- [x] Northshire Valley Spirit Healer 87047
- [x] Ironforge Spirit Healer 87044
- [x] Shadowglen Spirit Healer 87050

They are not sniffed so can't confirm if they are blizzlike or not, however they do not match up with facing data so its very likely they are in wrong position.

### CODE REVIEW

**Comment by Daribon** [comment]
Looks like I fried my brain yesterday and forgot to check 10 other guids. Will update once that is done.

**Comment by Daribon** [comment]
All that remains is getting data for the missing spirit healers in TODO list.

**Comment by Daribon** [comment]
All on the todo list completed, added the final spirit healers that weren't sniffed previously.

File: sql/migrations/20220705212140_world.sql [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+UPDATE `creature` SET  `position_x` = -14280.6, `position_y` = 300.899, `position_z` = 31.2034, `orientation` = 4.50295 WHERE `guid` = 40519;
+-- Correct position of Flame Crest Spirit Healer
+UPDATE `creature` SET  `position_x` = -7493.53, `position_y` = -2115.28, `position_z` = 136.081, `orientation` = 4.7473 WHERE `guid` = 40528;
+-- Correct position of Kargath Spirit Healer
+UPDATE `creature` SET  `position_x` = -6811.41, `position_y` = -2284.96, `position_z` = 280.836, `orientation` = 5.88176 WHERE `guid` = 40529;
+-- Correct position of Thelsamar Spirit Healer
+UPDATE `creature` SET  `position_x` = -5351.07, `position_y` = -2890.37, `position_z` = 341.881, `orientation` = 1.7693 WHERE `guid` = 40533;
+-- Correct position of Baradin Bay Spirit Healer
+UPDATE `creature` SET  `position_x` = -3341.49, `position_y` = -874.43, `position_z` = 1.05191, `orientation` = 1.93732 WHERE `guid` = 40535;
+-- Correct position of Southshore Spirit Healer
```
Comment (FlagFlayer) [inline sql/migrations/20220705212140_world.sql:28]: This has been pasted twice by mistake. This Spirit Healer is in Arathi Highlands.

Comment (Daribon) [reply sql/migrations/20220705212140_world.sql:28]: My bad. It's just the comment that is bad. The actual query changes the Arathi Highlands spirit healer.

**Pull request merged**